rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/project/ProjectController.java

```diff
@@ -576,9 +579,14 @@ public ResponseEntity<CollectionModel<EntityModel<Release>>> getReleasesOfLinked
     @RequestMapping(value = PROJECTS_URL + "/{id}", method = RequestMethod.DELETE)
     public ResponseEntity deleteProject(
             @Parameter(description = "Project ID")
-            @PathVariable("id") String id
+            @PathVariable("id") String id,
+            @Parameter(description = "Comment message")
+            @RequestParam(value = "comment", required = false) String comment
     ) throws TException {
         User sw360User = restControllerHelper.getSw360UserFromAuthentication();
+        if (!restControllerHelper.isWriteActionAllowed(projectService.getProjectForUserById(id, sw360User), sw360User) && comment == null) {
+            return new ResponseEntity<>(RESPONSE_BODY_FOR_MODERATION_REQUEST_WITH_COMMIT, HttpStatus.ACCEPTED);
```

Comment:
In case of missing request parameter, shouldn't the response status be `BAD_REQUEST`?